### ISSUE TITLE
Typing "n" breaks intellisense

### ISSUE THREAD

**Issue by Sharpiro** [PowerShell/vscode-powershell#193]
**Description**
When I type "n" on a fresh line, intellisense breaks until application is restarted

**Repo**
1. Open VS Code and create a new document
2. Select "Powershell" from the languages at the bottom right of the window
3. Type any command not starting with "n" to show that intellisense is working.
4. Type the letter "n" on a new line
5. Test to see that intellisense is no longer functioning
6. Restart VS Code (or close and re-open folder if you have one open) to regain functionality

**Ex:**
```
Test-Path
n
test-pat
```

**Notes**
- I updated to the latest version of VS Code yesterday
- I reinstalled vscode-powershell tools today
- New user, can't verify if this happened before

**Comment by daviwil** [comment]
Hey @Sharpiro, I just tried this with the latest releases of everything and couldn't reproduce it.  Could you  send some logs from the language server so that we can diagnose the issue?

You can gather logs by following these instructions:

Open up your VS Code User Settings (File -> Preferences -> User Settings) and insert this line into the file that appears in the pane on the right side:

`"powershell.developer.editorServicesLogLevel": "Verbose"`

Now restart VS Code and open a PowerShell file and try getting IntelliSense for a variable, then navigate to this path on your hard drive:

`c:\Users\<yourusername>\.vscode\extensions\ms-vscode.PowerShell-0.6.1\bin`

There should be a file there called EditorServices.log. Could you drag and drop it into your reply to this issue?

**Comment by Sharpiro** [comment]
So i tried reproducing this error on my personal machine and I could not do it either.

This is currently only occuring on my work machine.
Here are the logs for it:

[EditorServices.txt](https://github.com/PowerShell/vscode-powershell/files/275922/EditorServices.txt)

**Comment by rkeithhill** [comment]
This is the interesting bit:
```
    ---> (Inner Exception #0) System.ArgumentException: parsing "^\s*notepad++.exe\s+\[" - Nested quantifier +.
       at System.Text.RegularExpressions.RegexParser.ScanRegex()
       at System.Text.RegularExpressions.RegexParser.Parse(String re, RegexOptions op)
       at System.Text.RegularExpressions.Regex..ctor(String pattern, RegexOptions options, TimeSpan matchTimeout, Boolean useCache)
       at System.Text.RegularExpressions.Regex.IsMatch(String input, String pattern)
       at Microsoft.PowerShell.EditorServices.Protocol.Server.LanguageServer.CreateCompletionItem(CompletionDetails completionDetails, BufferRange completionRange, Int32 sortIndex)
```
I'm looking at it.  Doh.  This one is mine.  I should have regex escaped the tooltip text before injecting it into a regex pattern.  Well, at it should be easy to fix.

**Comment by Sharpiro** [comment]
any idea why this is happening sporadically, and what I could do for a workaround?

**Comment by rkeithhill** [comment]
Not yet but I can repro this exception so that should give me some confidence to fix it.

**Comment by rkeithhill** [comment]
@Sharpiro Do you have `notepad++.exe` on both machines?

**Comment by rkeithhill** [comment]
I'm closing this issue here and tracking it in PSES https://github.com/PowerShell/PowerShellEditorServices/pull/241

**Comment by Sharpiro** [comment]
I do have notepad++ on both machines, and the application installs to the machine's path by default I believe.

**Comment by rkeithhill** [comment]
On the machine that works, if you go to the PowerShell console and type `Get-Process -Name Note<tab>` does `notepad++.exe` show up as you tab through the process names?

**Comment by Sharpiro** [comment]
yea it does, this auto-populates
```
Get-Process -Name notepad++
```
```
Handles  NPM(K)    PM(K)      WS(K) VM(M)   CPU(s)     Id  SI ProcessName
-------  ------    -----      ----- -----   ------     --  -- -----------
    187      17     8016      17772   122     0.39   2604   1 notepad++
```

My work machine is Win7 while my home machine(the one with no issues) is Win10, both using Powershell 5, though it was manually installed on the Win7 box.

**Comment by rkeithhill** [comment]
Well I'm stumped for the moment.  I was able to repro on my Windows 10 machine running version`5.0.10586.122` of PowerShell.

**Comment by rkeithhill** [comment]
As for a workaround - how often do you use notepad++.exe?  :-)  The only workaround I know of is to either temporarily remove the notepad++ install dir from the path or rename notepad++.exe to maybe notepadpp.exe until we push out 0.6.2.

**Comment by Sharpiro** [comment]
removing it from path worked.  I can now type the letter "n" again lol.  Thanks.

**Comment by rkeithhill** [comment]
Sorry, that's kind of a crappy workaround (and my bad for introducing this bug in the first place) but that's all I got until we can push out updated bits.

**Comment by daviwil** [comment]
Since I've got another bug fix ready, I might be able to ship 0.6.2 on Monday.
